scipy/stats/__init__.py

```diff
@@ -569,6 +569,16 @@
    NearConstantInputWarning
    FitError
 
+Result classes used in :mod:`scipy.stats`
+-----------------------------------------
+
+.. warning:: These classes are private. Do not import them.
```

Comment:
One might wonder why they are documented at all then; perhaps we should explain. Also, rather than issuing a command to the user, consider phrasing it in terms of what SciPy will or will not support.

> These classes are private, but they are included here because instances of them are returned by other statistical functions. User import and instantiation is not supported.

Reply:
Let's do that 👍
```suggestion
.. warning::

   These classes are private, but they are included here because instances
   of them are returned by other statistical functions. User import and
   instantiation is not supported.
```